src/features/canvas/CanvasIntegration.ts

```diff
@@ -2,10 +2,9 @@
 // Initialize Canvas polling
 const CANVAS_API = 'https://sit.instructure.com/api/v1/';
 function getUpcomingAssignments() {
-    chrome.storage.local.get('tokens', tokens => {
-        // If you want to test this, uncomment the code below with your token
-        // tokens.canvas = '<YOUR TOKEN>';
-        if (!tokens.canvas) {
+    chrome.storage.local.get('tokens', object => {
+        let {tokens} = object;
```

Comment:
This seems weird. Would this not work?
```suggestion
    chrome.storage.local.get('tokens', {tokens} => {
```

Reply:
This would need to be the following to work (because parameters must be wrapped in parenthesis if they are not just a single parameter name):
```suggestion
    chrome.storage.local.get('tokens', ({tokens})=> {
```